xbanish.c

```diff
@@ -596,3 +605,17 @@ swallow_error(Display *d, XErrorEvent *e)
 
 	errx(1, "got X error %d", e->error_code);
 }
+
+int
+parse_cor(const char *s)
+{
+	int x, y;
+	if (s != NULL && sscanf(s, "x%dy%d", &x, &y) == 2) {
+		if (x < 0 || x > 100 || y < 0 || y > 100)
+			errx(1, "x and y must be between 0..100");
+		move_custom_x = x / 100.0f;
+		move_custom_y = y / 100.0f;
```

Comment:
Why not just let the X and Y values be anything and store them in `move_custom_x` and `move_custom_y` and then use those values for the absolute positioning in `hide_cursor`?

Reply:
Relative positioning will work regardless of the screen resolution, so `x50y100` always will be bottom-mid. Whereas absolute positioning will need to adjusted if the resolution changes.

I'm not using multi-monitor (with different resolutions) so I don't know how absolute positioning would affect those use-cases.

I can change it to absolute position if you prefer that.

Reply:
A value of `+50+100` will be absolutely positioned 50 pixels from the left and 100 pixels from the top.  A value of `-50-100` should place it 50 pixels from the right of the screen and 100 pixels from the bottom.